domain/src/main/java/org/oppia/android/domain/exploration/ExplorationProgressController.kt

```diff
@@ -619,6 +627,10 @@ class ExplorationProgressController @Inject constructor(
     }
   }
 
+  fun markContinueButtonAnimationSeen() {
```

Comment:
This isn't needed since the controller calls into profile management controller itself below.

Reply:
done